libpod/options.go

```diff
@@ -590,6 +590,18 @@ func WithNetNS(portMappings []ocicni.PortMapping) CtrCreateOption {
 	}
 }
 
+// WithLogPath sets the path to the log file
+func WithLogPath(path string) CtrCreateOption {
+	return func(ctr *Container) error {
+		if ctr.valid {
+			return ErrCtrFinalized
+		}
+		ctr.config.LogPath = path
```

Comment:
Can you add a check here that path != ""? I think a "" logpath will crash conmon

Reply:
If LogPath == "" then c.LogPath() will still generate the full logpath.  So it would make not difference.

Reply:
Hm. Can we move the != "" check here, and set the filepath.Join() default in NewContainer() instead of returning it from logPath()?

Reply:
Moved